docs/colors.mdx

```diff
@@ -1,11 +1,10 @@
 import { Meta } from "@storybook/addon-docs";
-import LinkTo from "@storybook/addon-links/react";
 
 <Meta title="Documentation/Colors" />
 
 # Colors
 
-Some components such as <LinkTo kind="Typography" story="page">Typography</LinkTo> have a `color`, `bg`/`backgroundColor` and `opacity` props, this functionality is provided by <a href="https://styled-system.com/api/#color">`styled-system/color`</a>.
+Some components such as [Typography](../?path=/docs/typography--docs) have a `color`, `bg`/`backgroundColor` and `opacity` props, this functionality is provided by [styled-system/color](https://styled-system.com/api/#color).

```

Comment:
**comment**: the "../" prefix was added here [to prevent Storybook from adding "iframe.html"](https://github.com/storybookjs/storybook/issues/8618#issuecomment-1401988950) to the start of the URL when the link is opened into a new tab.